openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -83,7 +84,7 @@ def create_mock_profile(self, user):
         legacy_profile = UserProfile.objects.get(id=user.id)
         legacy_profile.country = "US"
         legacy_profile.level_of_education = "m"
-        legacy_profile.year_of_birth = 1900
+        legacy_profile.year_of_birth = 2000
```

Comment:
I thought it was a more realistic value. :-) It's funny that this user is a male mother of twins born in 1900!